cli/cook/subcommands/usage.py

```diff
@@ -0,0 +1,218 @@
+import json
+import logging
+import sys
+
+from cook import http, colors
+from cook.format import format_job_memory
+from cook.querying import query_across_clusters, make_job_request
+from cook.util import guard_no_cluster, current_user, print_info, print_error
+
+
+def get_usage_on_cluster(cluster, user):
+    """Queries cluster for usage information for the given user"""
+    params = {'user': user, 'group_breakdown': 'true'}
+    usage_map = http.make_data_request(cluster, lambda: http.get(cluster, 'usage', params=params))
+    if not usage_map:
+        print_error(f'Unable to retrieve usage information on {cluster["name"]} ({cluster["url"]}).')
+        return {'count': 0}
+
+    share_map = http.make_data_request(cluster, lambda: http.get(cluster, 'share', params={'user': user}))
+    if not share_map:
+        print_error(f'Unable to retrieve share information on {cluster["name"]} ({cluster["url"]}).')
+        return {'count': 0}
+
+    settings_map = http.make_data_request(cluster, lambda: http.get(cluster, 'settings', params={}))
+    if not settings_map:
+        print_error(f'Unable to retrieve settings information on {cluster["name"]} ({cluster["url"]}).')
+        return {'count': 0}
+
+    resp = http.__get(f'http://{settings_map["mesos-master-hosts"][0]}:5050/redirect', allow_redirects=False)
+    if resp.status_code != 307:
```

Comment:
Is it worth still printing the cook usage data if we can't get the overall cluster utilization from mesos?

Reply:
Good call, done